### PR TITLE
[KOGITO-6654] SWF to use custom type

### PR DESCRIPTION
**Pull request by fjtirado**
Depends on https://github.com/kiegroup/kogito-runtimes/pull/1945

### CODE REVIEW

File: kogito-quarkus-examples/serverless-workflow-funqy/sw-funqy-workflow/src/main/resources/workflow.sw.json [hunk 1]
```diff
@@ -41,14 +41,17 @@
   "functions": [
     {
       "name": "countryFunction",
+      "type": "rest",
```
Comment (ricardozanini) [inline kogito-quarkus-examples/serverless-workflow-funqy/sw-funqy-workflow/src/main/resources/workflow.sw.json:44]: The default is rest. But just to make sure, the code is also defaulting to `rest`, right?

File: kogito-quarkus-examples/serverless-workflow-github-showcase/pr-checker-workflow/src/main/resources/handle-backend.sw.json [hunk 2]
```diff
@@ -7,19 +7,13 @@
...
     },
     {
       "name": "AddReviewers",
-      "metadata": {
-        "interface": "org.kogito.examples.sw.github.workflow.GitHubService",
-        "operation": "addReviewers",
-        "type": "service"
-      }
+      "type": "custom",
+      "operation": "service:java:org.kogito.examples.sw.github.workflow.GitHubService::addReviewers"
```
Comment (ricardozanini) [inline kogito-quarkus-examples/serverless-workflow-github-showcase/pr-checker-workflow/src/main/resources/handle-backend.sw.json:16]: Looks much better now! ❤️

**Review by ricardozanini — APPROVED** [review]

File: kogito-quarkus-examples/serverless-workflow-funqy/sw-funqy-workflow/src/main/resources/workflow.sw.json (hunk 1 above)
Comment (fjtirado) [reply kogito-quarkus-examples/serverless-workflow-funqy/sw-funqy-workflow/src/main/resources/workflow.sw.json:44]: yes, just for demo purposes, there is another example where I leave type empty

**Review by hbelmiro — APPROVED** [review]
LGTM.
Thank you @fjtirado.

**Comment by kie-ci1** [comment]
**(tests) - kogito-examples job** #437 was: **UNSTABLE**
Possible explanation: This should be test failures


**Test results:**
- PASSED: 432
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-examples.tests.kogito-examples/437/testReport/org.acme.travel.tests.multimessaging.quarkus/MultiMessagingIT/testProcess/">org.acme.travel.tests.multimessaging.quarkus.MultiMessagingIT.testProcess</a></summary>
org.opentest4j.AssertionFailedError: expected: <0> but was: <2>
	at org.acme.travel.tests.multimessaging.quarkus.MultiMessagingIT.testProcess(MultiMessagingIT.java:105)

</details>

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-examples.tests.kogito-examples/437/display/redirect

**Pull request merged**